CONTRIBUTING.md

```diff
@@ -64,7 +64,7 @@ Let's use `npm link` to symlink the fork into your project.
 
   module.exports = {
     // ..
-    // npm link mochapack hack
+    // yarn link mochapack hack
```

Comment:
Unclear to me if this hack is still needed. Updated it to be consistent, figured it can be deleted later if need be.

Reply:
It is needed. Otherwise webpack instance launched by mochapack will try to resolve loaders inside `mochapack/node_modules` and not inside your project `node_modules`. By providing `resolveLoader.root` option we are forcing webpack to resolve loaders inside the project's `node_modules`.